### PR TITLE
Add Pokémon Trainer Club

### PR DESCRIPTION
**Pull request by arche-dev**

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -8198,6 +8198,18 @@
             "points.com"
         ]
     },
+
```
Comment (tupaschoal) [inline _data/sites.json:8201]:
```suggestion

```

File: _data/sites.json [hunk 2]
```diff
@@ -8198,6 +8198,18 @@
             "points.com"
         ]
     },
+    
+    {
+        "name": "Pokémon Trainer Club",
+        "url": "https://support.pokemon.com/hc/en-us/articles/360000965826-How-do-I-delete-my-Pok%C3%A9mon-Trainer-Club-account-",
+        "difficulty": "easy",
+        "notes": "Go to https://club.pokemon.com/us/pokemon-trainer-club/edit-profile/ scroll down to the bottom abd click 'Request Data Deletion', enter the required fields and you'll then get a confirmation email.",
```
Comment (tupaschoal) [inline _data/sites.json:8206]:
```suggestion
        "notes": "Go to https://club.pokemon.com/us/pokemon-trainer-club/edit-profile/ scroll down to the bottom and click 'Request Data Deletion', enter the required fields and you'll then get a confirmation email.",
```
Is it deleted immediately?

Comment (arche-dev) [reply _data/sites.json:8206]: > Is it deleted immediately?

It's deleted when you click the button in the confirmation email

> Click the Delete Account button in the email. A confirmation page will display confirming that you have successfully deleted your Pokémon Trainer Club account.

**Review by tupaschoal — APPROVED** [review]
The entry looks fine, but there are merge conflicts, can you solve them please?

If you can't, I can do it in a few days

**Review by tupaschoal — COMMENTED** [review]
Small sorting error after merge:
"Keys must be in alphanumeric order. pokémon trainer club needs to come after pokémon go"

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**